client/web/web.go

```diff
@@ -35,8 +35,8 @@ import (
 type Server struct {
 	lc *tailscale.LocalClient
 
-	devMode   bool
-	loginOnly bool
+	devMode     bool
+	tsDebugMode string
```

Comment:
I don't entirely understand this change.  I get that you're adding debug flags for the new clients, but why does that replace loginOnly?  Are we going to need that later to know which mode we're running the client in?  Or is this just a temporary change and we'll switch it back later?

Reply:
Yeah we'll need it later to be piped through the CLI so we can set it for various platforms (or maybe even set it on by default). For now I wanted a way to specify the unreleased "login" or "full" modes, so rather than managing "login" through the opts and "full" via checking the debug flag, I figured we can just combine them onto `tsDebugMode` for now. Then pipe it through the CLI once we're ready.

Reply:
Sorry, one of my sentences was confusing...  I meant to ask, aren't we still going to need `loginOnly` later to indicate from the caller which mode we want the client to run in?  It would seem weird to rely on a debug flag to indicate that, even in non-debug usage.

Reply:
Yeah I think we will want it, but right now didn't want to expose it. So only using the debug flags until we're ready to release. Thoughts?

Reply:
I think that's reasonable